### DISCUSSION TITLE
migration from pages to app

### DISCUSSION THREAD

**Discussion by marco-digennaro** [vercel/next.js#44006] [Help]
hello, 
i migrated to app folder following the docs, i keep having a warning in the console:
`Warning: validateDOMNesting(...): <html> cannot appear as a child of <div>`

seems like **html** and **body** are injected twice after i added the _rootLayout_.
is there anything i'm missing or it's a bug?
i tried to download the beta and it seems working, but migrating gives me that warning

**Comment by ciruz** [comment]
Some kind of nesting error, do you have an example repo with the error?

**Comment by marco-digennaro** [comment]
i also thought so, but i removed all children till i got to the minimal structure:
`<html lang="en">
      <head />
      <body>{children}  </body>
    </html>`
    but it still gave me the warning.
    
i ended up cloning the latest canary and adding piece by piece there to get rid of it
